scripts/create_sfm_from_obj.py

```diff
@@ -0,0 +1,145 @@
+import os
+import sys
+import argparse
+from pathlib import Path
+import numpy as np
+import pycolmap
+import cv2
+from tqdm import tqdm
+from pixtrack.utils.pytorch3d_render_utils import (create_look_at_poses_for_mesh, 
+                                                   render_image,
+                                                   create_colmap_camera,
+                                                   create_colmap_image_from_pytorch3d_RT)
+from hloc import (extract_features, 
+                  match_features, 
+                  reconstruction, 
+                  pairs_from_exhaustive)
+import hloc.triangulation as triangulation
+from hloc.triangulation import (run_triangulation, 
+                                create_db_from_model,
+                                import_features, 
+                                import_matches, 
+                                geometric_verification)
+from hloc.reconstruction import (create_empty_db, 
+                                 import_images, 
+                                 get_image_ids)
+from hloc.utils.read_write_model import (write_images_binary, 
+                                         write_points3D_binary,
+                                         write_cameras_binary)
+
+
+def render_dataset_and_get_colmap_images_and_cameras(mesh_path, dataset_path, 
+                                                     fx, fy, cx, cy, W, H):
+    mesh_path = args.mesh_path
+    Rs, Ts, mesh = create_look_at_poses_for_mesh(mesh_path, subdivisions=2)
+    dataset_path.mkdir(parents=True, exist_ok=True)
+    mapping_path = dataset_path / 'mapping'
+    mapping_path.mkdir(parents=True, exist_ok=True)
+    colmap_images = {}
+    colmap_camera_id = 1
+    colmap_camera = create_colmap_camera(W, H, fx, cx, cy, k1=0.)
```

Comment:
pull the K out as a default arg to the func

Reply:
Done